.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ concurrency:
 jobs:
   test:
     # trunk-ignore(actionlint/runner-label)
-    runs-on: ubuntu-latest-32-cores
+    runs-on: ubuntu-latest-16-cores
```

Comment:
can't we just use ubuntu-latest then?

Reply:
Think we can, 16 cores is running in 12 minutes right now so will probably run in 25 minutes, should be ok